doc/faq.rst

```diff
@@ -80,6 +80,42 @@ from that location (i.e. ``myproject/binder/requirements.txt``) and will
 ignore those in the repository's root (``myproject/environment.yml`` and
 ``myproject/requirements.txt``).
 
+What factors influence how long it takes a Binder session to start?
+-------------------------------------------------------------------
+
+Understanding why some operations take longer than others requires a very
+brief overview of the pieces of machinery at play with BinderHub. There two
+things worth mentioning:
+
+* A *user pod* is the virtual machine that runs a users' code.
+* A *node* is the machine, running in the cloud, where a bunch of pods live.
+  There are many nodes for a Binder server, depending on the number of people
+  using the service.
```

Comment:
maybe add a third point: "Docker image store/registry, a binderhub uses one central place to store already built images. Shared amongst all the nodes in a cluster." or some such.